.github/workflows/stlab.yml

```diff
@@ -25,7 +22,6 @@ jobs:
 
       - name: Generate Job Matrix
         id: set-matrix
-        # Note: The json in this variable must be a single line for parsing to succeed.
```

Comment:
Is this no longer true? The comment originally motivated the need for `flatten_json.py`. I figured that was an unintuitive requirement of GH actions that merited a comment.

Reply:
Fixed